build.gradle

```diff
@@ -18,6 +18,17 @@ allprojects {
 }
 // Define versions in a single place
 ext {
+    // Sdk and tools
+    minSdkVersion = 19
+    targetSdkVersion = 28
+    compileSdkVersion = 28
```

Comment:
Probably worth to upgrade target and compile SDK versions to 29

Reply:
updated